cetmix_tower_server/models/cx_tower_reference_mixin.py

```diff
@@ -208,7 +214,14 @@ def copy(self, default=None):
         self.ensure_one()
         if default is None:
             default = {}
-        default["name"] = self._get_copied_name()
+
+        # skip copy name because this function use in models
+        # where it field name non store
+        default["name"] = (
```

Comment:
copy() function will use that original name automatically. No need to set it in code.